cupyx/scipy/sparse/csr.py

```diff
@@ -145,10 +145,12 @@ def __rtruediv__(self, other):
     # TODO(unno): Implement argmin
     # TODO(unno): Implement check_format
 
-    def diagonal(self):
+    def diagonal(self, k=0):
         # TODO(unno): Implement diagonal
         raise NotImplementedError
 
+    diagonal.__doc__ = base.spmatrix.diagonal.__doc__
+
```

Comment:
If its not Implemented, the docstring should reflect it too.
Just copying the base one is confusing.